src/ui/controls/input/index.tsx

```diff
@@ -0,0 +1,17 @@
+import {m} from 'malevic';
```

Comment:
I don't think there's a need in it, especially that it's hard to style it. `<input>` cannot have child nodes.